Makefile

```diff
@@ -122,30 +129,40 @@ push-tagged-image: export TAG=snapshot-$(shell git branch --show-current | sed "
 push-tagged-image: push-image
 
 # Generate manifests e.g. CRD, RBAC etc.
-manifests: manifests-k8s manifests-ocp
+manifests: prepare-manifests-directory manifests-k8s manifests-ocp strip-helm-labels
 
-manifests-k8s: manifests-crd manifests-k8s-csidriver
-	cp "$(KUBERNETES_CRD_AND_OTHERS_YAML)" "$(KUBERNETES_OLM_YAML)"
-	cat "$(KUBERNETES_CRD_AND_OTHERS_YAML)" "$(KUBERNETES_CSIDRIVER_YAML)" > "$(KUBERNETES_ALL_YAML)"
+strip-helm-labels:
```

Comment:
this should be done by checking `manifest` flag instead of removing them after the fact